tests/integrationTests/tests/python_simple_homework/__init__.py

```diff
@@ -0,0 +1,153 @@
+# Necessary imports. Provides library functions to ease writing tests.
+from lib import prebuild, testcase, SUBMITTY_INSTALL_DIR
+
+import subprocess
+import os
+import glob
+
+
+############################################################################
+# COPY THE ASSIGNMENT FROM THE SAMPLE ASSIGNMENTS DIRECTORIES
+
+SAMPLE_ASSIGNMENT_CONFIG = SUBMITTY_INSTALL_DIR + "/sample_files/sample_assignment_config/python_simple_homework"
+SAMPLE_SUBMISSIONS       = SUBMITTY_INSTALL_DIR + "/sample_files/sample_submissions/python_simple_homework"
+
+@prebuild
+def initialize(test):
+    try:
+        os.mkdir(os.path.join(test.testcase_path, "assignment_config"))
+    except OSError:
+        pass
+    try:
+        os.mkdir(os.path.join(test.testcase_path, "data"))
+    except OSError:
+        pass
+
+    subprocess.call(["cp",
+        os.path.join(SAMPLE_ASSIGNMENT_CONFIG, "config.h"),
+        os.path.join(test.testcase_path, "assignment_config")])
+    subprocess.call(["cp",
+        os.path.join(SAMPLE_ASSIGNMENT_CONFIG, "test_output", "hw01part1_sol.txt"),
+        os.path.join(test.testcase_path, "data")])
+
+
+############################################################################
+
+
+@testcase
+def correct(test):
+    return
```

Comment:
Are these tests broken (as several have return) or just something you left it by accident?